packages/utils/scripts/JestTransformer.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * https://babeljs.io/docs/en/config-files#jest
+ *
+ * Setup jest in Monorepo. This is to create a custom jest transformer file that wraps babel-jest.
+ *
+ * @type {Transformer}
+ */
+module.exports = require('babel-jest').createTransformer({
+  rootMode: 'upward',
+});
```

Comment:
Nice solution!! We'll probably want to move this in the root folder in case it's also required in other packages, but don't worry about this for now

Reply:
Good idea!